code/datums/shuttles.dm

```diff
@@ -372,6 +372,14 @@
 	description = "The Nanotrasen Emergency Shuttle Port(NES Port for short) is a shuttle used at other less known Nanotrasen facilities and has a more open inside for larger crowds, but fewer onboard shuttle facilities."
 	credit_cost = 500
 
+/datum/map_template/shuttle/emergency/rollerdome
+	suffix = "rollerdome"
+	name = "Uncle Pete's Rollerdome"
+	description = "Developed by a member of Nanotrasen's R&D crew that claims to have travelled from the year 2028. \
+	He says this shuttle is based off an old entertainment complex from the 1990s, though our database has no records on anything pertaining to that decade."
```

Comment:
uh, from a lore perspective, are you SURE that nanotrasen has no records on anything from the 90s? characters will often try to play off accidental/intentional references to real world events by saying they happened hundreds of years ago/during an era they're studying as a hobby, but if nanotrasen has no records whatsoever on an entire decade that's close to many modern day references, that could be a problem...

Reply:
I guess I can change it but I wasn't aware we were taking everything in the shuttle category as canon.